From 2cfea402d49d3ae28c9232f583eb95a3757c5186 Mon Sep 17 00:00:00 2001
From: zhengq <qi.zheng@intel.com>
Date: Wed, 27 May 2015 18:39:41 +0800
Subject: [PATCH] Pinctrl: Enable padmode switch for UART2 CTS.

For some GPS devices, the wake-up pin is UART2 CTS.
So when switching between GPIO and UART modes, we
need add the support for UART2 CTS.

Change-Id: Ic7d82a7f1dc1b3d6f8f88738a7214f73eb98ab61
Tracked-On: https://jira01.devtools.intel.com/browse/AKP-1733
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-37110
Signed-off-by: zhengq <qi.zheng@intel.com>
---
 drivers/pinctrl/pinctrl-cherryview.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pinctrl/pinctrl-cherryview.c b/drivers/pinctrl/pinctrl-cherryview.c
index 390bc94a7bb4..8b6199abaa7a 100644
--- a/drivers/pinctrl/pinctrl-cherryview.c
+++ b/drivers/pinctrl/pinctrl-cherryview.c
@@ -582,7 +582,7 @@ static struct gpio_pad_info southwest_pads_info[] = {
 	[UART2_RTS_B]		= { 1,		4,	-1, 1},
 	[UART1_TXD]			= { 1,		5,	-1, -1},
 	[UART2_TXD]			= { 1,		6,	-1, -1},
-	[UART2_CTS_B]		= { 1,		7,	-1, -1},
+	[UART2_CTS_B]		= { 1,		7,	-1, 1},
 	[23]				= { -1,		-1,	-1, -1},
 	[24]				= { -1,		-1,	-1, -1},
 	[25]				= { -1,		-1,	-1, -1},
-- 
1.9.1

